### ISSUE TITLE
[Feature Request] Avoid row down on ENTER when inline editing

### ISSUE THREAD

**Issue by tomaso909** [vitmalina/w2ui#1709]
When inline editing in the w2ui.grid the ENTER key applies the change and then selects the row below automatically. Is it possible to prevent this behavior and stay in the current row?
I tried this on the event onChange to try and re-select the original row but it does not work.
```
var changes = w2ui['grid'].getChanges()
w2ui['grid'].select(changes[0].recid);
```

**Comment by mpf82** [comment]
You can already do this by cancelling the edit event if the previous keycode was ENTER.

```
w2ui.grid.onEditField = function(event){
  if(event.originalEvent.keyCode === 13){
     event.preventDefault();
  }
};   
```

fiddle: http://jsfiddle.net/ty77umx1/1/

Please close the issue if the problem is solved, or leave it open if you still think this should be configurable in some other way.

Cheers =)

**Comment by tomaso909** [comment]
Just what I needed. Thank you!

**Comment by tomaso909** [comment]
This does not work in v2.0 since originalEvent is not available. I inspected the event object but could not see how to access the keyCode.

**Comment by tomaso909** [comment]
Solved it, in v2.0 this can be found inside "event.detail", therefore: event.detail.originalEvent.keyCode
